src/styles/utils/mixin.scss

```diff
@@ -0,0 +1,23 @@
+@mixin number-day {
+  @for $cell from 1 through 31 {
+    & :nth-child(#{$cell})::before {
+      content: '#{$cell}';
+    }
+  }
+}
+
+@mixin weak-day-lentgh {
```

Comment:
The mixin name 'weak-day-lentgh' seems to have a typo. It should be 'week-day-length' to accurately represent its purpose.